### ISSUE TITLE
Pop-up player playing at the same time as background (audio) player

### ISSUE THREAD

**Issue by assec42** [TeamNewPipe/NewPipe#2070]
- [x] I carefully read the [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/HEAD/.github/CONTRIBUTING.md) and agree to them.
- [x] I checked if the issue/feature exists in the latest version.
- [ ] I did use the [incredible bugreport to markdown converter](https://teamnewpipe.github.io/CrashReportToMarkdown/) to paste bug reports.
---------------------------------
So, I just installed the new version of NewPipe and wanted to see if the problems of the previous version were fixed. The first thing I did was play a video and then I switched to the background (music) player as it didn't work in 1.4.2 (while watching a video). The issue is that, in the settings, I enabled automatic pop-up player. Whenever I am switching to the background player while in a video, the pop-up player plays at the same time. This issue was not present in 1.4.1 and I hope it can be fixed.

Thank you very much for this wonderful application.
Regards.

**Comment by Skylead** [comment]
I had a similar bug to this when I just swapped to the automatic popup player on focus loss. Selecting background player from a main playback window causes the popup window to open the feed instead of background player (no longer dual playing at least it seems). I have noticed sometimes the background player gets half a word in before the popup player opens (and takes over from where it sounds like the main player left off and ignores any progress the background player made)

This seems to me like the dual playing bug is fixed, but there is a new one from the auto popup window being too aggressive. The auto popup should recognize playback changed due to user selection of playback mode and not think it's an application focus change.
